zarr/storage.py

```diff
@@ -1883,3 +1883,121 @@ def __delitem__(self, key):
         with self._mutex:
             self._invalidate_keys()
             self._invalidate_value(key)
+
+
+class ChunkCache(MutableMapping):
+    """Storage class that implements a least-recently-used (LRU) cache for array chunks.
+    Intended primarily for use with stores that can be slow to access, e.g., remote stores that
+    require network communication to store and retrieve data.
```

Comment:
Maybe add on the end of this sentence ", and/or arrays where decompression of data is computationally expensive.".